test/e2e/kubeproxy.go

```diff
@@ -420,10 +425,10 @@ func (config *KubeProxyTestConfig) setup() {
 	By("Getting ssh-able hosts")
 	hosts, err := NodeSSHHosts(config.f.Client)
 	Expect(err).NotTo(HaveOccurred())
-	if len(hosts) == 0 {
```

Comment:
NodeSSHHosts checks this:
```
if len(hosts) != len(nodelist.Items) {
```

We won't run this test on an empty cluster I guess.